src/auction.js

```diff
@@ -582,7 +582,9 @@ function tryAddVideoBid(auctionInstance, bidResponse, afterBidAdded, {index = au
     }), 'video');
   const context = videoMediaType && deepAccess(videoMediaType, 'context');
   const useCacheKey = videoMediaType && deepAccess(videoMediaType, 'useCacheKey');
+  const [hasTrackers, vastTrackers] = getVastTrackers(bidResponse);
```

Comment:
It's hard to work with functions that don't return standard types. In this case it appears to return an array where the first element is a bool, and the 2nd is an object. 
Instead, why not return null when there are no trackers ? The type is then a nullable object.
```
if (vastTrackers) { 
    bidResponse.vastXml = insertVastTrackers(vastTrackers, bidResponse.vastXml); 
}
```

Reply:
done